src/utils/markdownToolbar.jsx

```diff
@@ -82,7 +82,7 @@ export const orderedListButton = {
 }
 
 export const tableButton = {
-  name: "table",
+  name: "md-table",
```

Comment:
@seaerchin @kishore03109 @alexanderleegs This change is required as using "table" for name assigns it the className of table which has conflicting styles. See below screenshots for before and after. 

### Before

![Screenshot 2023-03-29 at 8 42 05 PM](https://user-images.githubusercontent.com/5507822/228539317-a0411da0-9ebe-4ee1-a57b-cd3f020e267d.png)

### After

![Screenshot 2023-03-29 at 8 42 36 PM](https://user-images.githubusercontent.com/5507822/228539349-837da062-b54c-460b-9c97-ab41669484a5.png)